### ISSUE TITLE
feedback link

### ISSUE THREAD

**Issue by maxf** [datagovuk/publish_data_alpha#375]
Decide how we want to gather feedback (email, form, zendesk) and implement feedback link.

**Comment by BlancaTortajada** [comment]
Does this need to be implemented for the assessment? If we only need to think about it, we could aim for a form like notify's. The good thing about it is that it helps prioritise. Could we link it to our support platform? here's notify's feedback form:

https://www.notifications.service.gov.uk/support

**Comment by gtdata** [comment]
yes, deffo need to scope this out. am finishing a feedback journey for registers so will share that with you and work out what we need for DGU
@BlancaTortajada @rossjones @maxf
